subt/teambase.py

```diff
@@ -9,9 +9,19 @@ def __init__(self, config, bus):
         super().__init__(config, bus)
         bus.register("broadcast")
 
+        self.robot_name = config.get('robot_name')
+        self.start_time = None  # unknown
+        self.finish_time = None  # infinite
+        if self.robot_name is not None:
+            self.finish_time = int(self.robot_name[1:].split('F')[0])  # T100 is accepted for example
```

Comment:
This seems unexpected. Why not the simpler
```suggestion
            self.finish_time = int(self.robot_name[1:])  # T100 is accepted for example
```